### DISCUSSION TITLE
Reduce CPU Usage

### DISCUSSION THREAD

**Discussion by sameer-coder** [emilk/egui#1261] [Q&A] [answered]
Hello!

Egui is an awesome project. I have recently started using rust and I am creating a GitHub Notifications manager in Egui to get more familiar with the framework. I was able to get a good looking and super responsive UI.

The problem is that even if I leave the app idle and minimize it, it still continues to show 1-2% CPU usage in reactive mode. It never really drops to zero. Continuous mode keeps the CPU usage at a constant 3%. I was wondering if there is anything I can do so that it stops using CPU when the app is idle. Any suggestions welcome!

**Comment by skydig** [comment, marked as answer]
have your tried latest main branch?
I pull latest code and 
under ubuntu, when I do not move mouse, the egui does not cosume CPU(I use top to watch)

**Reply by sameer-coder** [reply]
Indeed, I upgraded to the latest version and there is no cpu usage when idle. Thanks a lot!

**Comment by Areopagitics** [comment]
On the main site https://www.egui.rs/, if I open a few windows the cpu still goes up quite a bit.

**Reply by sameer-coder** [reply]
I think in the demo if you select "reactive mode" option the CPU drops. By default it uses continuous mode.
